src/DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -487,12 +487,12 @@ private function loadOdmDocumentManager(array $documentManager, ContainerBuilder
     /**
      * {@inheritdoc}
      */
-    protected function getMappingDriverBundleConfigDefaults(array $bundleConfig, \ReflectionClass $bundle, ContainerBuilder $container)
-    {
-        $this->bundleDirs[] = \dirname($bundle->getFileName());
-
-        return parent::getMappingDriverBundleConfigDefaults($bundleConfig, $bundle, $container);
-    }
+//    protected function getMappingDriverBundleConfigDefaults(array $bundleConfig, \ReflectionClass $bundle, ContainerBuilder $container)
+//    {
+//        $this->bundleDirs[] = \dirname($bundle->getFileName());
+//
+//        return parent::getMappingDriverBundleConfigDefaults($bundleConfig, $bundle, $container);
+//    }
 
     private function loadOdmDocumentManagerMappingInformation(array $documentManager, Definition $odmConfig, ContainerBuilder $container)
```

Comment:
```suggestion
    private function loadOdmDocumentManagerMappingInformation(array $documentManager, Definition $odmConfig, ContainerBuilder $container): void
```

as this is private, we could also add the void type for completeness.

Reply:
hm, then again there is a whole bunch of private methods, up to you if you want to cleanup that. we can also do that separately.

Reply:
Yes, I can send another PR for that but maybe keep it separate so we can keep the symfony 6 changes in this PR